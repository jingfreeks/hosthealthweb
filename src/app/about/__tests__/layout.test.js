import { render } from "@testing-library/react";
import AboutLayout from "../layout";

describe("About Layout Page", () => {
  it("renders About Layout Page to match snaptshot", () => {
    const { container } = render(<AboutLayout />);
    expect(container).toMatchSnapshot();
  });
});
