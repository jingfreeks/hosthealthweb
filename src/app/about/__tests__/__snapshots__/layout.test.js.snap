// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Layout Page renders About Layout Page to match snaptshot 1`] = `
<div>
  <nav>
    About Nav bar
  </nav>
  <main
    class="main"
  />
</div>
`;
